### PR TITLE
Add course info

### PR DESCRIPTION
**Pull request by joelostblom**
Let me know if this is in line with the latest changes to the syllabus and if you want another name for the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
-Information about BME1478H
+
+# BME1478H
+
+## Course description
+
+This course description can also be found on the [IBBME course page](https://ibbme.utoronto.ca/current-students/course-calendar/course-descriptions/).
+
+Today’s biomedical engineers will encounter many situations where an ability to perform basic computer programming is desirable if not essential. This is a hands-on course, teaching graduate students the basics of coding and data analysis in the context of different biomedical engineering scenarios. Students will become familiar with the UNIX operating environment, Python scripting, and task automation, in addition to analyzing, modelling, and manipulating data. The class will involve working through practical examples and solving real problems through a mix of online and live workshops. This course requires no previous programming experience.
+
```
Comment (linamnt) [inline README.md:9]: you can also mention "version control using Git and GitHub"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+## Course structure
+
+Each week students will be sent course materials on Monday or Tuesday. These will include a video tutorial, sample files, and a set of exercises. The course can be completed entirely online. Students will have opportunity to get one-on-one guidance from TAs in person once a week. Students will also be able to ask for help online using GitHub during.
+
```
Comment (linamnt) [inline README.md:13]: remove "using GitHub ~during~."?

**Review by linamnt — COMMENTED** [review]
Looks good other than the two comments.
I would also suggest including  a section called "Course Offerings" with the a link to the current class repo (i.e. Winter2020class) and then a list of Course Developers, TAs where we can list our names for that offering (so each course offering in the future can have a specific listing)

**Comment by joelostblom** [comment]
@linamnt Great suggestion!

**Review by linamnt — APPROVED** [review]

**Comment by SaraMati** [comment]
sorry to get to this late. do you want to add the content of the other repo as well (course description and learning outcomes): 
https://github.com/UofTCoders/grad-course/blob/master/proposal.md
https://github.com/UofTCoders/grad-course/blob/master/syllabus.md

**Pull request merged**